### DISCUSSION TITLE
[Turbopack] Support Config Similar to Webpack Resolve Fallback

### DISCUSSION THREAD

**Discussion by CHC383** [vercel/next.js#67196] [Ideas]
### Goals

1. Support config similar to Webpack [resolve.fallback](https://webpack.js.org/configuration/resolve/#resolvefallback)

### Background

There are cases where server-side modules are being imported but only be used in the server-side code, currently webpack config `resolve.fallback` is used to resolve "module not found" errors on the client-side code. See this [Stack Overflow post](https://webpack.js.org/configuration/resolve/#resolvefallback) for the details.

### Proposal

Add turbo.resolveFallback to allow modules not being imported on the client-side code.

**Comment by gobbodev** [comment]
nothing yet? ;(

**Comment by wallacerenan** [comment]
up

**Comment by timneutkens** [comment]
You can use something like the example shown here: https://nextjs.org/docs/app/api-reference/config/next-config-js/turbo#resolving-aliases but with an empty module, for example:

```
// empty.js
module.exports =  {}
``` 

```
module.exports = {
  experimental: {
    turbo: {
      resolveAlias: {
        fs: { browser: './empty.js' },
      },
    },
  },
}
```

Might add a specific option just for this, but that config should unblock you 🙂

**Reply by lcswillems** [reply]
Thank you! Yes, it would be great to have a config specifically for this :pray:
